scripts/zones/Rolanberry_Fields_[S]/mobs/Scabrous_Slug.lua

```diff
@@ -0,0 +1,17 @@
+-----------------------------------
+-- Area: Rolanberry Fields [S]
+--  Mob: Scabrous Slug
+-- Note: PH for Dyinyanga
+-----------------------------------
+local ID = zones[xi.zone.ROLANBERRY_FIELDS_S]
+-----------------------------------
+local entity = {}
+
+entity.onMobDeath = function(mob, player, optParams)
+end
+
+entity.onMobDespawn = function(mob)
+    xi.mob.phOnDespawn(mob, ID.mob.DYINYINGA_PH, 10, 3600) -- 1 hour
+end
```

Comment:
You could define the PH table as a local here in the mob script instead of setting a local variable in the IDs file.

Like so...
In the IDs file
```lua
    mob =
    {
        DYINYINGA = GetFirstID('Dyinyinga')
    }
```
And in this file
```lua
local dyinyingaPHTable =
{
    [ID.mob.DYINYINGA - 1] = ID.mob.DYINYINGA 
}

entity.onMobDeath = function(mob, player, optParams)
end

entity.onMobDespawn = function(mob)
    xi.mob.phOnDespawn(mob, dyinyingaPHTable, 10, 3600) -- 1 hour
end
```

Reply:
Is that how we'd like to do it? I thought we liked having the pH table in the zone table?

There's also the nms that have phs of different mob types

Reply:
I'll let others chime in, but I personally dont like local variables defined in the IDs files.

As an example, I do the same thing with Despot in this PR #5515

Reply:
I'll do whatever lsb team decides, but imo it's no different in terms of code cleanliness, but it does make the values part of the global table my way whereas your method doesn't

Reply:
Correct. Something that doesnt need to be global, shouldnt be global.